wrappers/services.go

```diff
@@ -201,12 +200,20 @@ func StartServices(apps []*snap.AppInfo, inter interacter, tm timings.Measurer)
 	return nil
 }
 
+type AddSnapServicesOptions struct {
+	Preseeding bool
+}
+
 // AddSnapServices adds service units for the applications from the snap which are services.
-func AddSnapServices(s *snap.Info, disabledSvcs []string, inter interacter) (err error) {
+func AddSnapServices(s *snap.Info, disabledSvcs []string, opts *AddSnapServicesOptions, inter interacter) (err error) {
```

Comment:
I wonder if we should put the disabledScvs under opts (maybe in a followup)?

Reply:
I haven't touched it because I think the service changes @stolowski has queued up remove it from here.

Reply:
Yes, that's correct, thanks for keeping this in mind.